editor/doc/editor_help.cpp

```diff
@@ -66,7 +66,7 @@
 #include "modules/mono/csharp_script.h"
 #endif
 
-#define CONTRIBUTE_URL vformat("%s/contributing/documentation/updating_the_class_reference.html", GODOT_VERSION_DOCS_URL)
+#define CONTRIBUTE_URL vformat("%s/documentation/class_reference/index.html", GODOT_VERSION_CONTRIBUTING_DOCS_URL)
```

Comment:
Not a blocker, but I'm unsure why this used `vformat` to begin with; concatenating adjacent literals was always supported:
```suggestion
#define CONTRIBUTE_URL GODOT_VERSION_CONTRIBUTE_URL "/documentation/class_reference/index.html"
```